gslib/commands/cp.py

```diff
@@ -282,82 +276,79 @@
     Copying file://obj [Content-Type=text/plain]...
     Uploading   gs://your-bucket/obj:                                182 b/182 B
 
-    If the checksum didn't match the service would instead reject the upload and
-    gsutil would print a message like:
+    If the checksums don't match, the service rejects the upload and
+    gsutil prints the following message:
 
     BadRequestException: 400 Provided MD5 hash "VgyllJgiiaRAbyUUIqDMmw=="
     doesn't match calculated MD5 hash "7gyllJgiiaRAbyUUIqDMmw==".
 
-  Even if you don't do this gsutil will delete the object if the computed
-  checksum mismatches, but specifying the Content-MD5 header has several
-  advantages:
+  Specifying the Content-MD5 header has several advantages:
 
-  1. It prevents the corrupted object from becoming visible at all, whereas
-     otherwise it would be visible for 1-3 seconds before gsutil deletes it.
+  1. It prevents the corrupted object from becoming visible. Otherwise, the
```

Comment:
Suggestion: Replace "otherwise" with "If you don't specify the header"

Reply:
Done.